app/Filament/Resources/UserResource.php

```diff
@@ -179,7 +181,8 @@ public static function table(Table $table): Table
                 Tables\Columns\TextColumn::make('ID')
                     ->label('ID')
                     ->searchable()
-                    ->sortable(),
+                    ->sortable()
+                    ->toggleable(isToggledHiddenByDefault: true),
```

Comment:
sounds good - changed in https://github.com/RetroAchievements/RAWeb/pull/2190/commits/7e6a875bb06228b7fe08ff6d5d6847e11bfe98f9

should I make it toggleable for achievements, too? this could then be the baseline for all the other upcoming resource, too.

Reply:
I'd recommend having it visible by default. As an admin, I'm most likely going to look for an achievement by ID than any other criteria.

Reply:
> i did not realize those toggles affect all other resources, too.

This feels weird. I don't see ID as toggleable on achievements. It's not visible by default. But if I toggle ID visibility on systems, it becomes visible on achievements!?

Reply:
good call, having IDs for systems and achievements visible at all times makes the most sense.
I removed toggleable() from all ID columns for now until I figure out how to not make it a global setting for all resources.

Reply:
> This feels weird. I don't see ID as toggleable on achievements. It's not visible by default. But if I toggle ID visibility on systems, it becomes visible on achievements!?

yeah, somehow that's connected globally. I'm not sure if this is intended by Filament's authors. I'll investigate and keep the ID column non-toggleable/always visible for now.

Reply:
Seems there's no easy way to have independent column visibility across resources.
Noted for future improvement.